frameworks/PHP/mako/benchmark_config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "framework": "mako",
+  "tests": [{
+    "default": {
+      "plaintext_url": "/plaintext",
+      "json_url": "/json",
+      "db_url": "/db",
+      "query_url": "/queries?queries=",
+      "fortune_url": "/fortunes",
+      "update_url": "/update?queries=",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Fullstack",
+      "database": "MySQL",
+      "framework": "mako",
+      "language": "PHP",
+      "flavor": "PHP8",
+      "orm": "raw",
```

Comment:
I think that it's using an ORM. It isn't using SQL direct.
I don't know if it is Micro or Full

Reply:
It's full. My bad. I copied this from Phalcon and even though they have a Full ORM, they are using "raw" :'(